lib/customerio/client.rb

```diff
@@ -2,6 +2,8 @@
 
 module Customerio
   class Client
+    VALID_PUSH_EVENTS = ['opened', 'converted', 'delivered']
```

Comment:
could be helpful to have these as const values in the module like `Customerio::PushOpened` as well